x/wasm-storage/types/errors.go

```diff
@@ -3,7 +3,14 @@ package types
 import "cosmossdk.io/errors"
 
 var (
-	ErrInvalidParam  = errors.Register("wasm-storage", 1, "invalid param")
-	ErrAlreadyExists = errors.Register("wasm-storage", 2, "already exists")
-	ErrInvalidLen    = errors.Register("wasm-storage", 3, "invalid len")
+	ErrInvalidParam          = errors.Register("wasm-storage", 1, "invalid param")
+	ErrAlreadyExists         = errors.Register("wasm-storage", 2, "already exists")
+	ErrInvalidFilterType     = errors.Register("wasm-storage", 3, "invalid filter type")
+	ErrInvalidFilterInputLen = errors.Register("wasm-storage", 4, "invalid filter input length")
+	ErrInvalidPathLen        = errors.Register("wasm-storage", 5, "invalid JSON path length")
+	ErrEmptyReveals          = errors.Register("wasm-storage", 6, "no reveals given")
+	ErrCorruptReveals        = errors.Register("wasm-storage", 7, "more than 1/3 of the reveals are corrupted")
+	ErrNoConsensus           = errors.Register("wasm-storage", 8, "less than 2/3 of the reveals in consensus range")
```

Comment:
```suggestion
	ErrNoConsensus           = errors.Register("wasm-storage", 8, "More than 1/3 of the reveals are not in consensus range")
```

Reply:
"More than or equal to 1/3 of the reveals are not in consensus range" right?

Reply:
True 🤣